content/applications/finance/payment_acquirers/paypal.rst

```diff
@@ -113,6 +90,48 @@ Your Paypal account is ready!
      `link <https://sandbox.paypal.com/cgi-bin/customerprofileweb?cmd=_profile-language-encoding>`_
      to configure the encoding format in a test environment.
 
+Settings in Odoo
+================
+
+.. seealso::
+   - :ref:`payment_acquirers/add_new`
+
+Credentials tab
+---------------
+
+Odoo needs your **API Credentials** to connect with your PayPal account. To do so, go to
+:menuselection:`Accounting --> Configuration --> Payment Acquirers` and :guilabel:`Activate` PayPal.
+Then, enter your PayPal account credentials in the :guilabel:`Credentials` tab:
+
+- :guilabel:`Email`: the login email address in Paypal;
+- :guilabel:`PDT Identity Token`: the key used to verify the authenticity of transactions;
+- :guilabel:`Use IPN`: whether you want to use **Instant Payment Notifications**.
+
+To set the :guilabel:`PDT Identity Token`, switch to :ref:`developer mode <developer-mode>` and
+retrieve the token by following the configuration steps at :ref:`paypal/pdt`.
+
+.. note::
+   The PayPal **Merchant ID** is not required.
+.. important::
+   If you are trying PayPal as a test, using a :ref:`PayPal Sandbox account <paypal/testing>`,
+   change the :guilabel:`State` to :guilabel:`Test Mode`. We recommend doing this on a test Odoo
+   database rather than on your main database.
+
+Fees tab
+--------
+
+You can charge **extra fees** to customers choosing to pay with PayPal in order to cover the
+transaction fees PayPal charges you. Once redirected to Paypal, your customer sees an extra amount
+applied to the order amount.
+
+To activate this, go to the :guilabel:`Fees` tab in Odoo and activate :guilabel:`Add Extra Fees`.
```

Comment:
Note: this should be replaced by a ref to [the new "Extra Fees" section in the parent page](https://www.odoo.com/documentation/16.0/applications/finance/payment_providers.html#extra-fees) starting from 16.0.